### PR TITLE
- add page with downloadable materials

### PR DESCRIPTION
**Pull request by EwelinaSkrzypacz**
In this pr, I added page with downloadable materials, like brand book and logo.

image

### CODE REVIEW

File: resources/views/logo.blade.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
+@extends("layout.public", ["excludeBigBubbles" => true])
+
+@php
+    $materials = [
+        [
+            'key' => 'brand_book',
+            'file_type' => 'pdf',
+            'file_size' => '2.4 MB',
+            'download_url' => url('materials/blumilk_brand_book_2024.pdf'),
```
Comment (mtracz) [inline resources/views/logo.blade.php:9]: Is it okey for us that file type and file size is added manually?

For file type we can use:
```php
$brandBookFilename = "blumilk_brand_book_2024.pdf";

$fileType =  Str:afterLast($brandBookFilename, ".");
 ```
 Or we can use PHP builtin function [mime_content_type](https://www.php.net/manual/en/function.mime-content-type.php)
 
 Ref:
 - https://laravel.com/docs/11.x/strings#method-str-after-last
 - https://www.php.net/manual/en/function.mime-content-type.php
 
 ---
 
 For filesize we can use PHP builtin function [filesize](https://www.php.net/manual/en/function.filesize.php) and convert it to MB.
 
 or[ Storage facade](https://laravel.com/docs/11.x/filesystem#file-metadata)

**Comment by mtracz** [comment]
PS. 
I think adding more readable commit messages than `wip` will help in code review understand what is going on in the current commit.

File: resources/views/logo.blade.php (hunk 1 above)
Comment (EwelinaSkrzypacz) [reply resources/views/logo.blade.php]: Fixed

**Comment by EwelinaSkrzypacz** [comment]
> PS. I think adding more readable commit messages than `wip` will help in code review understand what is going on in the current commit.

I will try

File: resources/views/logo.blade.php (hunk 1 above)
Comment (mtracz) [reply resources/views/logo.blade.php]: Will you replace `'` with `"`?

Comment (EwelinaSkrzypacz) [reply resources/views/logo.blade.php]: Done :)

**Review by mtracz — APPROVED** [review]

**Review by krzysztofrewak — APPROVED** [review]

**Pull request merged**